docs/crypto.md

```diff
@@ -2,6 +2,18 @@
 
 ## Adding Crypto
 
-The public releases of `aeacus` ship with very weak crypto. You should compile the binary for yourself after adding stronger crypto. This is not too hard, and almost all of the work is done for you.
+The public releases of `aeacus` ship with very weak crypto.
 
-**See the example in `aeacus-src/crypto.go`.** The project compiles as is, but again, you should read and change the file at least a little bit, then compile it for yourself.
+You can compile it yourself to generate random keys (`make release`). This means the public release decrypt function will not work.
+
+If security of the configuration is very important to you, or you feel the competition integrity is at risk, (e.g., you're running a competition with prizes, or running a practice session for reverse engineers), you should compile the binary for yourself after adding stronger crypto operations.
+
+This is not too hard, and almost all of the work is done for you. Anything you want to add is good! More XOR, AES with a hardcoded key, mixing bytes up... As long as the encrypt and decrypt functions work, nothing will break. The functions you would want to change are all in `crypto.go`.
```

Comment:
While I feel like the intentions here are good, trying to make it less daunting, this first sentence comes across as being a bit jerkish-- people who are just taking their first steps into Go through this SE (like I did) may end up very overwhelmed by compiling (with that said, you have documented compilation much more, which is helpful, but there are still random niche things that take place in Go development that not every newbie will be able to handle)
Is there any way this could be nice-ified a little?

Reply:
Good point, thanks for bringing it up! I rewrote it to hopefully be a bit nicer.